docs/internal/continuous-profiler.md

```diff
@@ -0,0 +1,255 @@
+between your process and the Collector.
+
+#### What if I'm on an unsupported .NET version?
+
+None of the .NET Framework versions is supported. You have to switch to
+supported .NET version.
+
+## Plugin
+
+For now, the plugins is responsible for
+
+* defining configuration for continuous profiling
+* providing exporter for the allocation and profiling data
+* *parsing data* prepared by the native code.
+
+### Plugin contract
+
+> [!IMPORTANT]  
+> It will be subject to change, when <https://github.com/open-telemetry/oteps/pull/239>
+> or <https://github.com/open-telemetry/oteps/pull/237> will be ready and merged.
+
+As other methods, `GetContinuousProfilerConfiguration` is called by reflection
+and convention.
+
+```csharp
+/// <summary>
+/// Configure Continuous Profiler.
+/// </summary>
+/// <returns>(threadSamplingEnabled, threadSamplingInterval, allocationSamplingEnabled, maxMemorySamplesPerMinute, exportInterval, continuousProfilerExporter)</returns>
+public Tuple<bool, uint, bool, uint, TimeSpan, object> GetContinuousProfilerConfiguration()
+{
+    var threadSamplingEnabled = true; // enables thread sampling
+    var threadSamplingInterval = 10000u; // interval to stop CLR runtime and fetch stacks. 10 000ms is Splunk default. 1000ms is the lowest supported value by Splunk. The code does not contains any limitations this. Plugins is responsible for checks.
+    var allocationSamplingEnabled = true; // enables allocation sampling
+    var maxMemorySamplesPerMinute = 200u; // max number of samples in minutes. 200 is tested default value by Splunk.
+    var exportInterval = TimeSpan.FromMilliseconds(500); // Pause time before next execution of exporting/reading buffer  process
+    object continuousProfilerExporter = new ConsoleExporter();
+    return Tuple.Create(threadSamplingEnabled, threadSamplingInterval, allocationSamplingEnabled, maxMemorySamplesPerMinute, exportInterval, continuousProfilerExporter);
+}
+```
+
+if more than one plugin implement `GetContinuousProfilerConfiguration` only
+the first one will be used. Other will be ignored.
+
+### Exporter contract
+
+Two methods has to be implemented by Exporter
+
+```csharp
+public void ExportThreadSamples(byte[] buffer, int read);
+public void ExportAllocationSamples(byte[] buffer, int read);
+```
+
+Both accept buffer produced by the native code together with the length of filled
+data. The Exporter is responsible both for parsing this buffer and exporting it.
+
+Example: ConsoleExporter in this PR.
+
+### Native parser
+
+As there is no default OpenTelemetry Protocol format there is not easy way to
+create good contract between OpenTelemetry Automatic Instrumentation and
+the plugin. The plugin has to implement (copy) our version of the parser.
+It should be changed when the OTel Proposal will be merged, and we can start implementing
+real OTLP exporter.
+
+Implementation can be found in `SampleNativeFormatParser` class in this PR.
```

Comment:
How is one supposed to use this information?

Reply:
Fixed in fb8de31a824fd491f737cdcb89656d99ab04190d. This PR should be merged after #3196